alacritty/build.rs

```diff
@@ -13,6 +13,6 @@
 // limitations under the License.
 
 fn main() {
-    let hash = rustc_tools_util::get_commit_hash().expect("couldn't get commit hash");
+    let hash = rustc_tools_util::get_commit_hash().unwrap_or("".to_owned());
```

Comment:
The correct method to use here is `unwrap_or_else(|| _)`. We should also switch `"".to_owned()` into something a little more clear like `String::new`:

```suggestion
    let hash = rustc_tools_util::get_commit_hash().unwrap_or_else(String::new);
```

Reply:
just `unwrap_or_default()`

Reply:
Absolutely correct, thanks!